src/mock/FakeHttpServer.h

```diff
@@ -0,0 +1,75 @@
+/* Copyright (c) 2022 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+
+#ifndef MOCK_FAKEHTTPSERVER_H_
+#define MOCK_FAKEHTTPSERVER_H_
+#include <vector>
+
+#include "proxygen/httpserver/HTTPServer.h"
+#include "proxygen/httpserver/RequestHandler.h"
+#include "proxygen/httpserver/RequestHandlerFactory.h"
+namespace nebula {
+
+class FakeHttpHandler : public proxygen::RequestHandler {
```

Comment:
Have a try with gmock?

Reply:
The libcurl is called in the httpclient. I can't mock libcurl, so I must access the network. In this case, I must need a FakeHttpServer. It can't be done through Gmock.